gitlab_api.py

```diff
@@ -14,6 +14,13 @@
 
 
 class GitlabAPI(object):
+    """
```

Comment:
This is a lint (quotes without any text).

The dockstring format should be:

```
"""One-line summary

Long description after a blank line, without indentation.
"""
```

Reply:
Done :ok_hand: